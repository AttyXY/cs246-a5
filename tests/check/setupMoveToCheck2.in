setup
+ K e5
+ k a1
+ B e1 
+ p h3
done
game human human
move e1 c3
move h3 h2
move c3 a1
quit 
quit
