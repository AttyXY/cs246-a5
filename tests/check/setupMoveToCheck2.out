WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING SETUP

8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  --------
1  --------
   abcdefgh

8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  --------
1  k-------
   abcdefgh

8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  --------
1  k---B---
   abcdefgh

8  --------
7  --------
6  --------
5  ----K---
4  --------
3  -------p
2  --------
1  k---B---
   abcdefgh

STARTING GAME!

8  --------
7  --------
6  --------
5  ----K---
4  --------
3  -------p
2  --------
1  k---B---
   abcdefgh

8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --B----p
2  --------
1  k-------
   abcdefgh

Black is in check.
8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --B----p
2  --------
1  k-------
   abcdefgh

Black is in check.
Invalid move. Input another move.
8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --B----p
2  --------
1  k-------
   abcdefgh

Black is in check.
Invalid move. Input another move.
Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
